governance.md

```diff
@@ -293,42 +294,27 @@ building proprietary products and services that leverage Project Software and
 Services. Funding acquired by Institutional Partners to work on The Project is
 called Institutional Funding. However, no funding obtained by an Institutional
 Partner can override The Project BDFL and Steering Council. If a Partner has
-funding to do Jupyter/IPython work and the Council decides to not pursue that
+funding to do Matplotlib work and the Council decides to not pursue that
 work as a project, the Partner is free to pursue it on their own. However in
 this situation, that part of the Partner’s work will not be under the
-Jupyter/IPython umbrella and cannot use the Project trademarks in a way that
+Matplotlib umbrella and cannot use the Project trademarks in a way that
 suggests a formal relationship.
 
-To acknowledge institutional contributions, there are two level of Institutional
-Partners, with associated benefits:
+To acknowledge institutional contributions, Institutional Partners
+receive the following benefits:
 
-**Tier 1** = an institution with at least one Institutional Council Member
-
--   Acknowledged on the Jupyter/IPython websites, in talks and T-shirts.
--   Ability to acknowledge their own funding sources on the Jupyter/IPython
+-   Acknowledged on the Matplotlib websites, in talks and T-shirts.
```

Comment:
I would interpret this as being talks _about_ matplotlib the library would fall under this, talks about process would not, and lightning talks are a gray area.  Any talk about mpl should acknowledge the (many) other contributors.   This should not be any more onerous than acknowledging your grants in scientific talks.

I suspect that we would end up with a stock image/slide with the proper set of logos.